CHANGELOG.md

```diff
@@ -13,3 +13,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Support for caching FactoryGirl factories when using ActiveRecord with Postgresql
 - User defined cache keys
 - Customizable cache timeout
+
+## [2.0.0] - 2018-10-17
+### Added
+
+- Support for FactoryBot namespace
```

Comment:
The changelog should also specify that FactoryGirl support is removed.

Reply:
good call, I'll add that in

Reply:
Thanks for working on this merge, @tmertens!

If you notice, we made it version 2.0.0 so that the '~>' syntax will work in a gemspec. If there is a need for a bug fix against the 1.x branch, you have plenty of room to cut a new 1.x release.